### DISCUSSION TITLE
Recruits that straddled two divisions

### DISCUSSION THREAD

**Discussion by lonniemc70** [ebzimny01/gd-recruit#52] [Ideas]
Is there away to identify recruits that show up in your division and also in the one above it using the gd-recruit tool?  In toying with the gd-recruit tool I found a recruit, using WIS Recruit Search, Rockne DIII,  Larry Weaver, QB, ID 23659335. He shows up in DII also, with same ID number and attributes. In gd-recruit tool he only shows up in DII, using the selection for my division and one higher.  WIS knowledge base speaks of such recruits and states that they may sign with you in the lower division based upon your teams success and your coaching abilities.  Presently I use a spreadsheet to compare DII/DIII recruits by position to find these recruits.  My team's success has improved quite a bit signing DII recruits.  If there was a command to only list higher division recruits that showed up in your division or show them as DII/DIII.  The gd-recruit tool only shows them in the higher division, which you cannot recruit from.

**Comment by ebzimny01** [comment]
Hi,
You are correct that when you select that checkbox it will grab recruits from one division higher.

However, the way WIS provides my application the recruiting data, it is not “customized” per coach unfortunately. All my app is able to do is indicate what WIS projects each recruits division to be. It does not take into consideration your recruiting “vision” as a coach.

For example, let’s say you are DIII coach and you have excellent recruiting “vision”. When you search for recruits using WIS website, you will see some recruits appear in your DIII search results that are DII caliber players. Now let’s say I am a DIII coach with normal recruiting “vision”. When I search WIS website for DIII recruits, I will not see some of those DII caliber players in my results. If I switched the search filter to DII, I would be able to see those players, but I would not likely be capable of signing any of them.

It’s hard to explain via email. If a verbal explanation would help, just give me a call. 561-289-5492


________________________________
From: lonniemc70 ***@***.***>
Sent: Monday, January 10, 2022 5:21:28 PM
To: ebzimny01/gd-recruit ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ebzimny01/gd-recruit] Recruits that straddled two divisions (Discussion #52)


Is there away to identify recruits that show up in your division and also in the one above it using the gd-recruit tool? In toying with the gd-recruit tool I found a recruit, using WIS Recruit Search, Rockne DIII, Larry Weaver, QB, ID 23659335. He shows up in DII also, with same ID number and attributes. In gd-recruit tool he only shows up in DII, using the selection for my division and one higher. WIS knowledge base speaks of such recruits and states that they may sign with you in the lower division based upon your teams success and your coaching abilities. Presently I use a spreadsheet to compare DII/DIII recruits by position to find these recruits. My team's success has improved quite a bit signing DII recruits. If there was a command to only list higher division recruits that showed up in your division or show them as DII/DIII. The gd-recruit tool only shows them in the higher division, which you cannot recruit from.

**Comment by lonniemc70** [comment]
Thanks for your reply. I was just wondering if there was a comparing command or line of instruction that could be incorporated (without bogging down the program) that could eliminate higher division recruits (regardless of vision) that you have no chance whatsoever of signing. I am not a programmer, but I know syntax has to make sense to the language used.  I do not know if your code extracts higher division first then the division you are in, so I am thinking if lower division is extracted first and then the higher division. As each higher division recruit is extracted and then compared to lower division recruits, if there is no match it is skipped. Only matches would be selected. Vision or customization would not be needed per coach, as I believe your program pulls all recruits from the division your are assigned to in WIS plus the one above it if that option is selected. 

Just a thought and trying not to do extra work that a computer could do.

Thanks,

Lonnie McArthur






On Monday, January 10, 2022, 04:58:34 PM CST, ebzimny01 ***@***.***> wrote: 






Hi,
You are correct that when you select that checkbox it will grab recruits from one division higher.

However, the way WIS provides my application the recruiting data, it is not “customized” per coach unfortunately. All my app is able to do is indicate what WIS projects each recruits division to be. It does not take into consideration your recruiting “vision” as a coach.

For example, let’s say you are DIII coach and you have excellent recruiting “vision”. When you search for recruits using WIS website, you will see some recruits appear in your DIII search results that are DII caliber players. Now let’s say I am a DIII coach with normal recruiting “vision”. When I search WIS website for DIII recruits, I will not see some of those DII caliber players in my results. If I switched the search filter to DII, I would be able to see those players, but I would not likely be capable of signing any of them.

It’s hard to explain via email. If a verbal explanation would help, just give me a call. 561-289-5492


________________________________
From: lonniemc70 ***@***.***>
Sent: Monday, January 10, 2022 5:21:28 PM
To: ebzimny01/gd-recruit ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ebzimny01/gd-recruit] Recruits that straddled two divisions (Discussion #52)


Is there away to identify recruits that show up in your division and also in the one above it using the gd-recruit tool? In toying with the gd-recruit tool I found a recruit, using WIS Recruit Search, Rockne DIII, Larry Weaver, QB, ID 23659335. He shows up in DII also, with same ID number and attributes. In gd-recruit tool he only shows up in DII, using the selection for my division and one higher. WIS knowledge base speaks of such recruits and states that they may sign with you in the lower division based upon your teams success and your coaching abilities. Presently I use a spreadsheet to compare DII/DIII recruits by position to find these recruits. My team's success has improved quite a bit signing DII recruits. If there was a command to only list higher division recruits that showed up in your division or show them as DII/DIII. The gd-recruit tool only shows them in the higher division, which you cannot recruit from.

—
Reply to this email directly, view it on GitHub<https://github.com/ebzimny01/gd-recruit/discussions/52>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AHVOUMCIDMRFRUUKP3MA2P3UVNLWRANCNFSM5LUZFSKQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ebzimny01** [comment]
Simply put, my app only has access to the default division level assigned by WIS to each recruit.

The only way you would know for sure if you have a chance of signing the recruit is if you go directly to the website and search your division for that recruit. If you are in DIII and search for DIII recruits, the players that show up there are ones you have a shot at signing.
________________________________
